aip/general/0133.md

```diff
@@ -53,7 +53,7 @@ rpc CreateBook(CreateBookRequest) returns (Book) {
   - The collection's parent resource **should** be called `parent`, and
     **should** be the only variable in the URI path.
   - The collection identifier (`books` in the above example) **must** be
-    literal.
+    literal string.
```

Comment:
Shouldn't that be "a literal string" ? As in the change is missing the indefinite article "a"?

Reply:
A thanks